<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExpSignalProcessTool</class>
 <widget class="QWidget" name="ExpSignalProcessTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="6" column="1">
        <widget class="QPushButton" name="pushButton_plotSingleRFData_PreDomain">
         <property name="text">
          <string>单通道RF数据频域绘制</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_loadRFData">
         <property name="text">
          <string>导入RF数据</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_plotAllRFData">
         <property name="text">
          <string>RF数据预览</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QToolButton" name="toolButton_confirmSelectedSensorIndex">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>导入方式</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="spinBox_selectedSensorIndex">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox_RFDataLoadType"/>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="pushButton_plotSingleRFData">
         <property name="text">
          <string>单通道RF数据时域绘制</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>选择单通道</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_preProcessRFData">
         <property name="text">
          <string>预处理RF数据</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_plotAllPreProcessRFData">
         <property name="text">
          <string>预处理后的RF数据预览</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_savePreProcessRFData">
         <property name="text">
          <string>保存预处理后的RF数据</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButton_plotSinglePreProcessedRFData">
         <property name="text">
          <string>单通道预处理后RF数据时域绘制</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="pushButton_plotSingleBeforeAndAfterRFData">
         <property name="text">
          <string>单通道预处理前后时域对比绘制</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="pushButton_plotSingleBeforeAndAfterRFData_FreDomain">
         <property name="text">
          <string>单通道预处理前后频域对比绘制</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="pushButton_plotSinglePreProcessedRFData_FreDomain">
         <property name="text">
          <string>单通道预处理后RF数据频域绘制</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="horizontalSpacer_10">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="7" column="7">
      <widget class="QCheckBox" name="checkBox_isHilbertTransform">
       <property name="text">
        <string>希尔伯特变换处理</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>置零时间步长</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>滤波方式</string>
       </property>
      </widget>
     </item>
     <item row="5" column="6">
      <widget class="QToolButton" name="toolButton_setLouKongProcessPara">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QToolButton" name="toolButton_setFilterPara">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QCheckBox" name="checkBox_isFilter">
       <property name="text">
        <string/>
       </property>
       <property name="tristate">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QSpinBox" name="spinBox_cutZerosTimesteps">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>4096</number>
       </property>
       <property name="value">
        <number>800</number>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <widget class="QCheckBox" name="checkBox_isCutZeros">
       <property name="text">
        <string/>
       </property>
       <property name="tristate">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QComboBox" name="comboBox_deConvolutionType">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="5" column="7">
      <widget class="QCheckBox" name="checkBox_isLouKong">
       <property name="text">
        <string/>
       </property>
       <property name="tristate">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QComboBox" name="comboBox_insertValueType">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QComboBox" name="comboBox_filterType">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QCheckBox" name="checkBox_isDeconvolution">
       <property name="text">
        <string/>
       </property>
       <property name="tristate">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>镂空阵列处理设置</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>去卷积方式</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QToolButton" name="toolButton_setDeconvolutionPara">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>插值方式</string>
       </property>
      </widget>
     </item>
     <item row="6" column="7">
      <widget class="QCheckBox" name="checkBox_isInsertValue">
       <property name="text">
        <string/>
       </property>
       <property name="tristate">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="6">
      <widget class="QToolButton" name="toolButton_setInsertValuePara">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="8">
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="8">
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="8">
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="8">
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
